### DISCUSSION TITLE
Help, Am professor in university

### DISCUSSION THREAD

**Discussion by samirpops** [community/community#127383] [GitHub Education]
Hello, Am professor in French université,
my proof paper, scanned, are not recognized by the automatic system.
Help.
So I can't use teacher profil in github academy
I have all proofs that I am professor
Regards

**Comment by imohitpatel** [comment]
Wait for sometime github community will review your university documents and also approach to your colleague who have github teacher account they will help you and suggest the process of getting verified

**Reply by samirpops** [reply]
Thanks for return, I received email telling me that the verification is failed !
